src/bin/server/accounts/account.rs

```diff
@@ -14,7 +14,7 @@ pub struct Account {
     balance: u64,
 }
 
-const INITIAL_BALANCE: u64 = 10;
+const INITIAL_BALANCE: u64 = 100000;
```

Comment:
Could this be part of the configuration?

Reply:
Hum, not the local configuration, it's a network wide one as every node needs to have the same value.
We might put it as a config for the EPFL faucet (when/if that's done).